snippets/rust.cson

```diff
@@ -42,13 +89,27 @@
       \t$3
       }
     '''
+  "if let … = … { … }":
+    prefix: "if-let"
+    body: '''
+     if let ${1:Some(pat)} = ${2:expr} {
+       ${2:unimplemented!();}
+     }
+   '''
   'if':
     'prefix': 'if'
     'body': '''
       if ${1:expression} {
       \t$2
       }
     '''
+  "impl … for … { … }":
+    prefix: "impl-trait"
```

Comment:
When `impl Trait` feature will be implemented, the name will conflict with the feature.
I'd suggest `impl-trait-for`.